Shopify/cli
# Show status message when watching files during function replay

### WHY are these changes introduced?

When using `function replay` in watch mode, it's not obvious that the process is waiting for file changes.

### WHAT is this pull request doing?

This PR adds a status message when watching files during `function replay` so that it's obvious that the process didn't hang, and that it's just waiting for file changes.

### How to test your changes?

- Run `function replay` in a function directory
- See that the message is printed

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
